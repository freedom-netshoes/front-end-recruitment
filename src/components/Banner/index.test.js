import Banner from './';

describe('Banner', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Banner />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
