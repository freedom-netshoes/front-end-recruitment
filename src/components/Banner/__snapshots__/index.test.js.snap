// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner should render correctly 1`] = `
<div
  className="main-banner"
>
  <img
    alt="TimÃ£o"
    src="http://localhost:8001/static/images/banner.png"
  />
</div>
`;
